src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_collectbuildenv.yaml

```diff
@@ -0,0 +1,56 @@
+post:
+  summary: Collect build environment information.
+  description: |
+    Creates _buildenv files based on origin package builds. This can be used
+    to re-use exact older build enviroment even when further new binary packages
+    got added. For example to re-build an old maintenance update in the same way.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: oproject
+      required: true
+      schema:
+        type: string
+      description: Origin project. The project that the Origin Package will have the build information copied from.
+    - in: query
+      name: opackage
+      required: true
+      schema:
+        type: string
+      description: Origin package. Build environment information of the package that will be copied.
+    - in: query
+      name: orev
+      schema:
+        type: string
+      description: Origin package revision. Revision of the Origin Package on which to base the build environment collection.
+    - in: query
+      name: comment
+      description: Comment for the new revision.
+  responses:
+    '200':
+      description: OK
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/revision.yaml'
```

Comment:
The current apidocs states that the answer follows the status.xsd schema, [here](https://build.opensuse.org/apidocs/index#64). I haven't done any test of this enpoint, but, does it really answer with a revision element?

I have checked we don't have a revision schema, only the revisionlist schema. In case the response follows the status schema, we should mention the schema used in the `description:` field.

Reply:
I see. Fine.